### ISSUE TITLE
Its not working with long delay

### ISSUE THREAD

**Issue by vivekdoshi2** [lamphej/OpenVBX-Delayed-Response#1]
I have setup the plugin and working fine for short time period like 15min or 30min

But when I tried for long time say 24hours or 36 hours,I dont get any sms

Please help me with this.

**Comment by zurez** [comment]
I am facing the same issue . Doesnt work for 24 hours.
Vivek what is the maximum delay you were able to do ?

**Comment by vivekdoshi2** [comment]
Its working for 3 to 4 hours.

**Comment by zurez** [comment]
For me its only working for 45 minutes. Could it be a server /hosting issue?

**Comment by vivekdoshi2** [comment]
Yes I think so,I have made the changes just testing if I got that than I will put the changes list here.

**Comment by zurez** [comment]
Maybe setting the execution time to infinity may do the trick. Btw which hosting are you on ?
